cirq_superstaq/ops/qubit_gates.py

```diff
@@ -246,71 +254,111 @@ def _num_qubits_(self) -> int:
+    def _value_equality_values_(self) -> Tuple[cirq.TParamVal, cirq.TParamVal]:
+        if cirq.is_parameterized(self.theta):
+            theta = self.theta
+        else:
+            theta = self.theta % (2 * np.pi)
+
         if cirq.is_parameterized(self.sandwich_rx_rads):
-            return self.polarity, self.sandwich_rx_rads
+            sandwich_rx_rads = self.sandwich_rx_rads
+        else:
+            sandwich_rx_rads = self.sandwich_rx_rads % (4 * np.pi)
 
-        return self.polarity, self.sandwich_rx_rads % (4 * np.pi)
+        return theta, sandwich_rx_rads
 
-    def _value_equality_approximate_values_(self) -> Tuple[str, cirq.PeriodicValue]:
-        return self.polarity, cirq.PeriodicValue(self.sandwich_rx_rads, 4 * np.pi)
+    def _value_equality_approximate_values_(self) -> Tuple[cirq.PeriodicValue, cirq.PeriodicValue]:
+        return (
+            cirq.PeriodicValue(self.theta, 2 * np.pi),
+            cirq.PeriodicValue(self.sandwich_rx_rads, 4 * np.pi),
+        )
 
     def _equal_up_to_global_phase_(self, other: Any, atol: float) -> Optional[bool]:
         if not isinstance(other, AceCR):
             return NotImplemented
 
-        if other.polarity != self.polarity:
-            return False
-
-        return approx_eq_mod(self.sandwich_rx_rads, other.sandwich_rx_rads, 2 * np.pi, atol=atol)
+        return approx_eq_mod(self.theta, other.theta, 2 * np.pi, atol=atol) and approx_eq_mod(
+            self.sandwich_rx_rads, other.sandwich_rx_rads, 2 * np.pi, atol=atol
+        )
 
     def __repr__(self) -> str:
-        if not self.sandwich_rx_rads:
-            return f"css.AceCR({self.polarity!r})"
-        return f"css.AceCR({self.polarity!r}, {self.sandwich_rx_rads!r})"
+        if not self.sandwich_rx_rads and self.theta == np.pi / 2:
+            return "css.AceCR"
+        elif self.theta == np.pi / 2:
+            return f"css.AceCR(sandwich_rx_rads={self.sandwich_rx_rads!r})"
+        elif not self.sandwich_rx_rads:
+            return f"css.AceCR(theta={self.theta})"
+        return f"css.AceCR(theta={self.theta}, sandwich_rx_rads={self.sandwich_rx_rads!r})"
 
     def __str__(self) -> str:
-        if not self.sandwich_rx_rads:
-            return f"AceCR{self.polarity}"
-        return f"AceCR{self.polarity}|{cirq.rx(self.sandwich_rx_rads)}|"
+        if not self.sandwich_rx_rads and self.theta == np.pi / 2:
+            return "AceCR"
+        elif not self.sandwich_rx_rads:
+            return f"AceCR({self.theta})"
+        elif self.theta == np.pi / 2:
+            return f"AceCR|{cirq.rx(self.sandwich_rx_rads)}|"
+        return f"AceCR({self.theta})|{cirq.rx(self.sandwich_rx_rads)}|"
 
     def _json_dict_(self) -> Dict[str, Any]:
-        return cirq.obj_to_dict_helper(self, ["polarity", "sandwich_rx_rads"])
+        if self.theta != np.pi / 2:
+            return cirq.obj_to_dict_helper(self, ["sandwich_rx_rads", "theta"])
+        return cirq.obj_to_dict_helper(self, ["sandwich_rx_rads"])
```

Comment:
no need for the special case here
```suggestion
        return cirq.obj_to_dict_helper(self, ["sandwich_rx_rads", "theta"])
```